specification/ACKNOWLEDGEMENTS.adoc

```diff
@@ -0,0 +1,34 @@
+= Acknowledgements
```

Comment:
I noticed that Acknowledgements is not an appendix in the glTF spec: https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#acknowledgments

Might be good to understand why to see if we should do the same thing.

Reply:
This seems to be a bit arbitrary. From a quick (!) search, Vulkan does not seem to contain Acknowledgements at all. OpenCL seems to insert them at the top (before all sections, but unnumbered. apparently with the AsciiDoc `[[acknowdlegements]]` section type). I'll take a note to investigate that further.

Reply:
Ok, the current approach is fine then